Yosemite/YosemiteTests/Stores/JustInTimeMessageStoreTests.swift

```diff
@@ -0,0 +1,132 @@
+import XCTest
+import TestKit
+
+@testable import Yosemite
+@testable import Networking
+
+final class JustInTimeMessageStoreTests: XCTestCase {
+
+    /// Mock Network: Allows us to inject predefined responses!
+    ///
+    private var network: MockNetwork!
+
+    /// Mock Storage: InMemory
+    ///
+    private var storageManager: MockStorageManager!
+
+    /// Testing SiteID
+    ///
+    private let sampleSiteID: Int64 = 123
+
+    var sut: JustInTimeMessageStore!
```

Comment:
Not really important, but just curious, why is this named `sut`?

Reply:
I think it refers to "subject under test"?